02.general/06.userscripts/docs.en.md

```diff
@@ -7,55 +7,71 @@ visible: true
 ---
 
 * [Introduction](#intro)
-* [Where can you get userscripts?](#repo)
-* [Userstyles.org](#userstyles)
-* [Greasy Fork](#greasyFork)
-* [OpenUserJS.org](#openUserJs)
+    * [Supported apps](#products)
+    * [Adguards recommended scripts](#scripts)
+* [Where to get more userscripts](#repo)
+    * [Userstyles.org](#userstyles)
+    * [Greasy Fork](#greasyfork)
+    * [OpenUserJS.org](#openuserjs)
+    * [Community](#community)
 
-## Introduction
 
 <a name="intro"></a>
+## Introduction
 
-Starting from version 5.9 it became possible to install userscripts in AdGuard for Windows.
+First, let's get it clear. What is a userscript? Userscripts (we also call them 'extensions' sometimes) de-facto are mini-programs written in Javascript. They modify or extend functionality of one or more websites. Many AdGuard users must be already familiar with such userscripts as AdGuard Assistant, PopupBlocker and AdGuard Extra(#extra).
 
-First of all, we need to understand what it is. Userscript is a script written in Javascript that extends the functionality of one or more sites. AdGuard users already know about such extensions. Among them are the [PopupBlocker](https://github.com/AdGuardTeam/PopupBlocker) that blocks all unwanted pop-up windows, and the [AdGuard Assistant](https://github.com/AdGuardTeam/adguardassistant) which helps to manage filtering right from the browser. You can see its icon in the corner of any web page.
+<a name="products"></a>
+## Supported apps
 
-As it was mentioned, AdGuard for Windows can work as a userscript manager. The detailed information about how to install userscripts in AdGuard you will find in the [Extensions](https://kb.adguard.com/en/windows/features/extensions) article of our Knowledgebase.
+Currently, custom scripts are supported in: AdGuard for Windows, AdGuard for Android and AdGuard for Mac (Mac version does not yet allow you to add your own custom scripts).
 
+<a name=-"scripts></a>
+### AdGuard userscripts
 
+>### ATTENTION
+>These userscripts come directly from AdGuard developers and we can guarantee that they will be effective and safe. Keep in mind that whenever you download a userscript from an unknown source you expose yourself to a risk. Some scripts can be harmful to your computer. Always get new scripts only from sources you trust 100%. Examples of popular sites with scripts - [trustworthy sources](#repo).
```

Comment:
Further are [some examples](#repo) of popular sites with scripts.

Reply:
*